js/formValidation.js

```diff
@@ -0,0 +1,26 @@
+const emailValidationRegex = /^[a-z]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$/;
+const email = document.getElementById('mail');
+const submitForm = document.querySelector('#contact-form');
+const errorMessage = document.querySelector('.error-message');
+
+function checkEmail(emailInput) {
+  if (emailInput.trim() === '') {
+    return false;
+  }
+  if (!emailValidationRegex.test(emailInput)) {
+    return false;
+  }
+  return true;
+}
+
+submitForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const emailValue = email.value;
+  if (checkEmail(emailValue)) {
+    errorMessage.style.display = 'none';
+    submitForm.submit();
+  } else {
+    errorMessage.style.display = 'block';
+    errorMessage.textContent = 'Please enter a valid email';
+  }
+});
```

Comment:
[OPTIONAL] I think we could wrap all of our JS code in a `main` function. And the call `main` once the web app is finished loading. This will ensure that it works correctly. Imagine that the script loads, but maybe some of the elements on the document aren't still ready. Then it could fail to select these.

Start the script running once the page has finished loading like this: `window.onload = main`